.gitignore

```diff
@@ -24,3 +24,4 @@ multi_ec2.yaml
 *.egg-info
 .eggs
 cover/
+roles/infra-ansible/
```

Comment:
Do we still need this line?

Reply:
I think this is a strict dependency module for configuring DNS infra.